{"uuid": "b60fdb47-7526-4496-84a5-8b6be05acc06", "children": ["03de902c-c26c-48c8-ba24-5f0f68b938b1"], "befores": [{"name": "logged_in_page", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to be visible\nActual value: <element(s) not found> \nCall log:\n  - Expect \"to_be_visible\" with timeout 15000ms\n  - waiting for get_by_alt_text(\"国語\")\n\n", "trace": "  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\DELL\\pytest-sample\\tests\\conftest.py\", line 67, in logged_in_page\n    expect(page.get_by_alt_text(\"国語\")).to_be_visible(timeout=15000)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 20401, in to_be_visible\n    self._sync(self._impl_obj.to_be_visible(visible=visible, timeout=timeout))\n    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ~~~~~~~~~~~^^\n  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 763, in to_be_visible\n    await self._expect_impl(\n    ...<5 lines>...\n    )\n  File \"C:\\Users\\DELL\\pytest-sample\\.venv\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 83, in _expect_impl\n    raise AssertionError(\n        f\"{out_message}\\nActual value: {actual} {format_call_log(result.get('log'))}\"\n    )\n"}, "start": 1756459668849, "stop": 1756459690981}], "start": 1756459668849, "stop": 1756459690989}